### ISSUE TITLE
Getting Emergency Brake Vehicle List

### ISSUE THREAD

**Issue by amin-tabrizian** [eclipse-sumo/sumo#13922]
Hi, 

Is there any way to get the list of all vehicles that performed emergency braking?

traci.simulation.getEmergencyStoppingVehiclesIDList() will get the list of emergency stopped vehicles not emergency braking ones.

**Comment by m-kro** [comment]
No, there is none. Out of curiosity: What exactly is your use case?

**Comment by amin-tabrizian** [comment]
I am working on a merging scenario and want to optimize the behavior of the merging vehicle using some feedbacks from the environment. One of the components of the environment feedback is the list of vehicles which performed emergency braking as a result of the merging vehicle’s dangerous behavior.
